docs/src/modules/java/pages/workflows.adoc

```diff
@@ -217,42 +216,31 @@ include::example$transfer-workflow-compensation/src/main/java/com/example/transf
 
 Design for failure is one of the key attributes of all Akka components. Workflow has the richest set of configurations from all of them. It's essential to build robust and reliable solutions.
 
-=== Timeouts
-
-By default, a workflow run has no time limit. It can run forever, which in most cases is not desirable behavior. A workflow step, on the other hand, has a default timeout of 5 seconds. Both settings can be overridden in the workflow `configuration` method.
+=== Step Timeouts
 
-[source,java,indent=0]
-.{sample-base-url}/transfer-workflow-compensation/src/main/java/com/example/transfer/application/TransferWorkflow.java[TransferWorkflow.java]
-----
-include::example$transfer-workflow-compensation/src/main/java/com/example/transfer/application/TransferWorkflow.java[tag=timeouts]
-----
-<1> Sets a timeout for the duration of the entire workflow. When the timeout expires, the workflow is finished and no transitions are allowed.
-<2> Sets a default timeout for all workflow steps.
+Workflows do not have a global timeout and can run indefinitely. Only step timeouts are supported.
```

Comment:
Do we even have to mention that?

Perhaps repeat what is mentioned above instead:
```
If you need a global workflow timeout, consider using durable timers as described in the xref:#_timeouts[Timeouts] section.
```

Btw, that xref is obsolete now?

Reply:
Ok, I will re-work it. The sentence "If you need a global timeout..." do not belong to the "Pausing workflow". 

We should do the other way around.

Reply:
@patriknw, changed here cbd2903d